app/views/directives/logs/_log-viewer.html

```diff
@@ -68,14 +68,13 @@
 <div ng-show="state=='logs'">
   <div class="log-view" id="{{logViewerID}}" ng-class="{'log-fixed-height': fixedHeight}">
```

Comment:
Just noticed this, for dynamic id on line 69, are we supposed to use `ng-attr-id`?  Its not part of your PR (and I bet I overlooked this initially) so prob not a big deal.